checkout.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FITBIT</title>
+
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+        integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous" />
+
+    <link rel="stylesheet" href="https://suspicious-sammet-33f1b3.netlify.app/Components/components.css" />
+    <link rel="stylesheet" href="mycart.css" />
+    <link rel="stylesheet" href="common.css" />
+    <link rel="stylesheet" href="checkout.css" />
+</head>
+
+<body>
+    <!--  ******************************** Navbar Start ********************************  -->
+    <nav class="navbar">
+        <div>
+            <img alt="hamburger-icon" src="images/hamburger.svg" class="hamburger-icon" id="hamburger-icon" />
+        </div>
+        <div class="logo-container">
+            <a class="logo" href="index.html">
+                <img class="logo-image" alt="logo-image" src="images/neatUI.svg" />
+                <span class="logo-text">FiT</span>
+                BiT
+            </a>
+        </div>
+        <div class="nav-items-container-end">
+            <div class="login-search-bar-container">
+                <a href="logout.html" class="button button-primary">Logout</a>
+                <div class="input-box">
+                    <input type="text" class="input" value="" onchange="change(this)" required />
```

Comment:
value attribute can be removed if not required

Reply:
Ok i will see this